packages/babel-jest/src/index.js

```diff
@@ -69,8 +74,7 @@ const createTransformer = (options: any) => {
   options = Object.assign({}, options, {
     plugins: (options && options.plugins) || [],
     presets: ((options && options.presets) || []).concat([jestPreset]),
-    retainLines: true,
```

Comment:
`retainLines` needs to remain for now. We'll look into removing it after getting this merged